contributing-guides/alias-pages.md

```diff
@@ -0,0 +1,300 @@
+# Alias pages
+
+In order to document a command which is an alias of another command, you can
+write an alias page. This file contains a list of all translations of the alias
+page template decided upon in
+[#5368](https://github.com/tldr-pages/tldr/pull/5368).
+The templates can be changed when necessary.
+
+[en](#en) •
+[bs](#bs) •
+[da](#da) •
+[de](#de) •
+[es](#es) •
+[fa](#fa) •
+[fr](#fr) •
+[hi](#hi) •
+[id](#id) •
+[it](#it) •
+[ja](#ja) •
+[ko](#ko) •
+[ml](#ml) •
+[nl](#nl) •
+[no](#no) •
+[pl](#pl) •
+[pt_BR](#pt_br) •
+[pt_PT](#pt_pt) •
+[ru](#ru) •
+[sh](#sh) •
+[sv](#sv) •
+[ta](#ta) •
+[th](#th) •
+[tr](#tr) •
+[zh](#zh) •
+[zh_TW](#zh_tw)
+
+---
+### en
+
+```markdown
+# example
+
+> This command is an alias of `example`.
+
+- View documentation for the original command:
+
+`tldr example`
+```
+
+---
+### bs
+
+```markdown
+# example
+
+> Ova komanda je alijas za `example`.
+
+- Pogledaj dokumentaciju za izvornu komandu:
+
+`tldr example`
+```
+
+---
+### da
+
+Not translated yet.
+
+---
+### de
+
+```markdown
+# example
+
+> Dieser Befehl ist ein Alias von `example`.
+
+- Zeige die Dokumentation für den originalen Befehl an:
+
+`tldr example`
+```
+
+---
+### es
+
+```markdown
+# example
+
+> Este comando es un alias de `example`.
+
+- Ver documentación para el comando original:
+
+`tldr example`
+```
+
+---
+### fa
+
+Not translated yet.
+
+---
+### fr
+
+```markdown
+# example
+
+> Cette commande est un alias de `example`.
+
+- Voir la documentation de la commande originale :
+
+`tldr example`
+```
+
+---
+### hi
+
+```markdown
+# example
+
+> यह आदेश `example` का उपनाम है।
+
+- मूल आदेश के लिए दस्तावेज़ देखें:
+
+`tldr example`
+```
+
+---
+### id
+
+```markdown
+# example
+
+> Perintah ini merupakan alias dari `example`.
+
+- Menampilkan dokumentasi untuk perintah asli:
+
+`tldr example`
+```
+
+---
+### it
+
+Not translated yet.
+
+---
+### ja
+
+```markdown
+# example
+
+> このコマンドは `example` のエイリアスです.
+
+- オリジナルのコマンドのドキュメントを表示する:
+
+`tldr example`
+```
+
+---
+### ko
+
+Not translated yet.
+
+---
+### ml
+
+```markdown
+# example
+
+> ഈ കമാൻഡ് `example` എന്നത്തിന്റെ അപരനാമമാണ്.
+
+- യഥാർത്ഥ കമാൻഡിനായി ഡോക്യുമെന്റേഷൻ കാണുക:
+
+`tldr example`
+```
+
+---
+### nl
+
+```markdown
+# example
+
+> Dit commando is een alias van `example`.
+
+- Bekijk de documentatie van het originele commando:
+
+`tldr example`
+```
+
+---
+### no
+
+```markdown
+# eksempel
```

Comment:
```suggestion
# example
```

Reply:
Totally forgot about that, thanks for noticing